values_hybrid.yaml

```diff
@@ -22,27 +20,40 @@ redis:
       - "redis.tyk.svc.cluster.local:6379"
     useSSL: false
     #If you're using Bitnami Redis chart please input your password in the field below
+    #pass: ""
 
 gateway:
   kind: DaemonSet
   # Set replicaCount for Deployments
   replicaCount: 2
-  tags: "ingress"
+  # if enableSharding set to true then you must define a tag to load APIs to these gateways i.e. "ingress"
+  tags: ""
   hostName: ""
   tls: true
   containerPort: 8080
-  # For MDCB setups change to connection string for your MDCB instance
+  # For on premise MDCB setups sipmly change the connection string to point to your MDCB instance
   rpc:
-    connString: "hybrid.cloud.tyk.io:9091"
+    # On the Tyk Cloud Console find this value in the endpoints for your control plane deployment
+    connString: "MDCB-INGRESS:443"
```

Comment:
What's the reasoning to change this default to a dummy value?

Reply:
New cloud users need to take precedence and every single person i have interacted with so far assumes that `hybrid.cloud.tyk.io:9091` will magically work for them with Ara. 

I would rather they have to read the comment and think - if they need the classic cloud ingress i would prefer they use the whole commented block underneath with that configuration

Reply:
I can just make it blank for now